x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -87,6 +87,12 @@ const HOST_ISOLATION_EXCEPTIONS_LABELS: ArtifactListPageProps['labels'] = Object
       defaultMessage: '"{itemName}" has been removed from host isolation exception list.',
       values: { itemName },
     }),
+  emptyStateTitleNoEntries: i18n.translate(
+    'xpack.securitySolution.hostIsolationExceptions.emptyStateTitleNoEntries',
+    {
+      defaultMessage: 'There are no host isolation exception entries to display.',
```

Comment:
Should this be `There are no host isolation exceptions to display.` instead?

Reply:
🦅 👁️ !  d080743